python/pyarrow/_compute.pyx

```diff
@@ -950,6 +1132,16 @@ cdef class _FilterOptions(FunctionOptions):
 
 
 class FilterOptions(_FilterOptions):
+    """
+    Options for selecting with a boolean filter.
+
+    Parameters
+    ----------
+    null_selection_behavior : str, default "drop"
+        How to handle nulls in the input.
```

Comment:
```suggestion
        How to handle nulls in the selection filter.
```

(the `filter` docstring uses "input" for the values that are being filtered)

Reply:
Will do, thanks.